src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
+        for deployment_config in self.list_deployment_configs():
+            config_names.append(deployment_config.get("ConfigName"))
+            instance_types.append(deployment_config.get("BenchmarkMetrics").key()[0])
+
+            for benchmark_metric in deployment_config.get("BenchmarkMetrics").values():
+                if benchmark_metric["name"] == "Latency":
+                    latencies.append(benchmark_metric["value"])
+                if benchmark_metric["name"] == "Throughput":
+                    through_puts.append(benchmark_metric["value"])
+                if benchmark_metric["name"] == "Cost":
+                    instance_rates.append(benchmark_metric["value"])
+
+        df = pd.DataFrame(
+            {
+                "Config Name": config_names,
+                "Min. Latency<br>(Ms/Token)": latencies,
+                "Max. Throughput<br>(Tokens/Second)": through_puts,
+                "Instance Type": instance_types,
+                "Instance Rate<br>($/Hour)": instance_rates,
+            }
+        )
+
+        df["Instance Rate<br>($/Hour)"] = df["Instance Rate<br>($/Hour)"].apply(
+            lambda rate: "${0:.2f}".format(float(rate))
+        )
+        headers = {"selector": "th", "props": "text-align: left; width: 10%"}
+        df = (
+            df.style.set_caption("Benchmark Metrics")
+            .set_table_styles([headers])
+            .set_properties(**{"text-align": "left"})
+        )
+
+        display(df)
+
+    def list_deployment_configs(self) -> List[Dict[str, Any]]:
+        """List deployment configs for ``This`` model in the current region."""
+        if self.__deployment_configs is None:
+            self.__deployment_configs = get_jumpstart_configs(
+                region=self.region,
+                model_id=self.model_id,
+                model_version=self.model_version,
+                sagemaker_session=self.sagemaker_session,
+            )
+
+        configs = []
+        for config_name in self.__deployment_configs.keys():
+            jumpstart_metadata_config = configs.get(config_name)
+            configs.append(
+                {
+                    "ConfigName": config_name,
+                    "BenchmarkMetrics": jumpstart_metadata_config.benchmark_metrics,
```

Comment:
nit: add a type conversion helper or extend the `JumpStartMetadataConfig` class to have a new method which returns the desired schema